src/lib.rs

```diff
@@ -1,27 +1,30 @@
 // For the full copyright and license information, please view the LICENSE
 // file that was distributed with this source code.
 
-// Expose parse_datetime
-pub mod parse_datetime;
-
-use chrono::{Duration, Local, NaiveDate, Utc};
-use regex::{Error as RegexError, Regex};
+use regex::Error as RegexError;
 use std::error::Error;
 use std::fmt::{self, Display};
 
+// Expose parse_datetime
+pub mod parse_relative_time;
+
+use chrono::{DateTime, FixedOffset, Local, LocalResult, NaiveDateTime, TimeZone};
+
+use crate::parse_relative_time::parse_relative_time;
+
 #[derive(Debug, PartialEq)]
-pub enum ParseDurationError {
+pub enum ParseDateTimeError {
     InvalidRegex(RegexError),
     InvalidInput,
 }
 
-impl Display for ParseDurationError {
+impl Display for ParseDateTimeError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            ParseDurationError::InvalidRegex(err) => {
+            ParseDateTimeError::InvalidRegex(err) => {
```

Comment:
```suggestion
            Self::InvalidRegex(err) => {
```
There was a clippy lint for this in the coreutils repo. Why should probably enable that here too.

Same goes for the other match arm.